src/nodes.js

```diff
@@ -104,41 +108,42 @@ var FromImport = Node.extend("FromImport", {
                     names || new NodeList(), withContext);
     }
 });
-var FunCall = Node.extend("FunCall", { fields: ['name', 'args'] });
-var Filter = FunCall.extend("Filter");
-var FilterAsync = Filter.extend("FilterAsync", {
+var FunCall = Node.extend('FunCall', { fields: ['name', 'args'] });
+var Filter = FunCall.extend('Filter');
+var FilterAsync = Filter.extend('FilterAsync', {
     fields: ['name', 'args', 'symbol']
 });
-var KeywordArgs = Dict.extend("KeywordArgs");
-var Block = Node.extend("Block", { fields: ['name', 'body'] });
-var Super = Node.extend("Super", { fields: ['blockName', 'symbol'] });
-var TemplateRef = Node.extend("TemplateRef", { fields: ['template'] });
-var Extends = TemplateRef.extend("Extends");
-var Include = TemplateRef.extend("Include");
-var Set = Node.extend("Set", { fields: ['targets', 'value'] });
-var Output = NodeList.extend("Output");
-var TemplateData = Literal.extend("TemplateData");
-var UnaryOp = Node.extend("UnaryOp", { fields: ['target'] });
-var BinOp = Node.extend("BinOp", { fields: ['left', 'right'] });
-var In = BinOp.extend("In");
-var Or = BinOp.extend("Or");
-var And = BinOp.extend("And");
-var Not = UnaryOp.extend("Not");
-var Add = BinOp.extend("Add");
-var Sub = BinOp.extend("Sub");
-var Mul = BinOp.extend("Mul");
-var Div = BinOp.extend("Div");
-var FloorDiv = BinOp.extend("FloorDiv");
-var Mod = BinOp.extend("Mod");
-var Pow = BinOp.extend("Pow");
-var Neg = UnaryOp.extend("Neg");
-var Pos = UnaryOp.extend("Pos");
-var Compare = Node.extend("Compare", { fields: ['expr', 'ops'] });
-var CompareOperand = Node.extend("CompareOperand", {
+var KeywordArgs = Dict.extend('KeywordArgs');
+var Block = Node.extend('Block', { fields: ['name', 'body'] });
+var Super = Node.extend('Super', { fields: ['blockName', 'symbol'] });
+var TemplateRef = Node.extend('TemplateRef', { fields: ['template'] });
+var Extends = TemplateRef.extend('Extends');
+var Include = TemplateRef.extend('Include');
+var Set = Node.extend('Set', { fields: ['targets', 'value'] });
+var Output = NodeList.extend('Output');
+var TemplateData = Literal.extend('TemplateData');
+var UnaryOp = Node.extend('UnaryOp', { fields: ['target'] });
+var BinOp = Node.extend('BinOp', { fields: ['left', 'right'] });
+var In = BinOp.extend('In');
+var Or = BinOp.extend('Or');
+var And = BinOp.extend('And');
+var Not = UnaryOp.extend('Not');
+var Add = BinOp.extend('Add');
+var Sub = BinOp.extend('Sub');
+var Mul = BinOp.extend('Mul');
+var Div = BinOp.extend('Div');
+var FloorDiv = BinOp.extend('FloorDiv');
+var Mod = BinOp.extend('Mod');
+var Pow = BinOp.extend('Pow');
+var Neg = UnaryOp.extend('Neg');
+var Pos = UnaryOp.extend('Pos');
+var Compare = Node.extend('Compare', { fields: ['expr', 'ops'] });
+var CompareOperand = Node.extend('CompareOperand', {
     fields: ['expr', 'type']
 });
 
-var CustomTag = Node.extend("CustomTag", {
```

Comment:
Ah, you are correct, it must be leftover from the initial custom tags work. `CallExtension` is the node that this one became. We can safely remove it.